### PR TITLE
Added Anime.js example

### PR DESCRIPTION
**Pull request by lkwatson**
Anime.js example! Uses CSS translation, and completion callbacks.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,10 +40,12 @@ <h1 class="title">Bouncy Ball</h1>
     <label for="smil" class="label label-smil">SMIL</label>
     <input type="radio" id="mojs" name="approaches" />
     <label for="mojs" class="label label-mojs">mo.js</label>
+    <input type="radio" id="anime" name="approaches" />
+    <label for="anime" class="label label-anime">anime.js</label>
   </form>
   <div class="panes">
     <div class="docs-toggle">
-      <button class="docs-toggle-link button-link" id="toggle">about <span class="demo-name">Vanilla JS</span></button>
+      <a class="docs-toggle-link" id="toggle" href="#">about <span class="demo-name">Vanilla JS</span></a>
```
Comment (bryanbraun) [inline index.html]: I think we want to keep this as a button. This was a recent change, and I'm guessing it got mixed up in a rebase.

File: examples/anime/script.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+var bounceDown = anime({
+  autoplay: false,    //See similar comments above
+  targets:  '#ball',
+  translateY: {
+    value:    ['0px', '160px'], //When bouncing down, start at 0px and end at 160px
+    duration: 575,
+    easing:   'easeInQuad',
+  },
+  direction: 'normal',
+  loop:      false,
```
Comment (bryanbraun) [inline examples/anime/script.js]: I think we can reduce the amount of code here by removing properties that use their default value. For example, `loop: false` and `direction: 'normal'` are default. I think they can be removed from both functions.

File: examples/anime/index.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  <meta charset="UTF-8">
+  <title>Bouncy Ball - Comparing Web Animation Techniques - anime.js</title>
+  <link rel="stylesheet" href="../shared-styles.css">
+  <style>
+    #ball {
+      display: block;
+      width: 50px;
+      height: 50px;
+      border-radius: 50%;
+      background-color: #ae27e7;
```
Comment (bryanbraun) [inline examples/anime/index.html]: Nice color! Can we also add it to the bottom of [this list of link colors in the CSS](https://github.com/sparkbox/bouncy-ball/blob/65673f33b75d49fbf415f5ce1d74b2e4c0266cae/site/css/styles.css#L140-L149)? This is the button text color for when the demo is selected. Currently, it just show black text on a dark background 😞 :

image

File: index.html (hunk 1 above)
Comment (lkwatson) [inline index.html]: You're right, this got mixed up in the rebase.

File: examples/anime/script.js (hunk 2 above)
Comment (lkwatson) [inline examples/anime/script.js]: Will do, that makes sense. My thoughts in including it was just for clarity's sake.

File: examples/anime/readme.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+Anime (/ˈæn.ə.meɪ/) is a flexible yet lightweight JavaScript animation library.
+It works with CSS, Individual Transforms, SVG, DOM attributes and JS Objects.
+
+Anime.js License: MIT
+
+## Project Links
+
+* [Anime.js Github Repo](https://github.com/juliangarnier/anime)
+* [Anime.js examples on Codepen](http://codepen.io/collection/XLebem/)
```
Comment (bryanbraun) [inline examples/anime/readme.md]: Thanks for adding some docs here. 🙌

File: examples/anime/script.js (hunk 2 above)
Comment (bryanbraun) [inline examples/anime/script.js]: I think we can reduce the amount of code here by removing properties that use their default value. For example, `loop: false` and `direction: 'normal'` are default, so I think they can be removed from both functions.

File: index.html (hunk 1 above)
Comment (bryanbraun) [inline index.html]: We want to keep this as a button. This was a recent change, and I'm guessing it got mixed up in a rebase.

**Comment by bryanbraun** [comment]
Looks great! Thanks again for working on this. :shipit:

**Comment by lkwatson** [comment]
No problem! Thank you.

**Pull request merged**